cdap-common/bin/common.sh

```diff
@@ -267,6 +267,10 @@ cdap_set_hive_classpath() {
     # classpath from it and export it to use it in the launch command
     if [ -n "${HIVE_HOME}" -a -n "${HIVE_CONF_DIR}" -a -n "${HADOOP_CONF_DIR}" ]; then
       EXPLORE_CONF_FILES=$(ls -1dF ${HIVE_CONF_DIR}/* ${HADOOP_CONF_DIR}/* | sed -e '/\/$/d' | tr '\n' ':')
+      if [ -n "${TEZ_HOME}" ]; then
```

Comment:
This assumes that TEZ_HOME is in the environment. I don't see that getting set, anywhere, and it isn't showing up in the Hive environment.
```
$ export | grep TEZ
$ grep TEZ_HOME /etc/hadoop/conf/* 2>/dev/null
$ hive -e 'set -v' 2>&1 | tr ' ' '\n' | sed -e 's/amb37.*net/REDACTED/' | grep TEZ
tez.tez-ui.history-url.base=http://REDACTED:8080/#/main/views/TEZ/0.7.0.2.3.4.0-70/TEZ_CLUSTER_INSTANCE
$ rpm -qa | grep tez
tez_2_3_4_0_3485-0.7.0.2.3.4.0-3485.el6.noarch
```